content/tutorials/quickstart/broker-sale.md

```diff
@@ -0,0 +1,1053 @@
+---
+html: broker-sale.html
+parent: xrpl-quickstart.html
+blurb: Broker a sale between a sell offer and a buy offer.
+labels:
+  - Accounts
+  - Quickstart
+  - Broker
```

Comment:
At some point we should add landing pages and colors for these labels (they'll be unclickable gray labels as is). I can push a commit that does that later though.